third_party/pcsc-lite/naclport/server_clients_management/src/known_client_apps.json

```diff
@@ -101,5 +101,9 @@
   "chrome-extension://gpnmjjdkmdfmccnpepmianiomfahpbcc": {
     "name": "Ujambo Kiosk App (development)",
     "autoapprove": true
+  },
+  "chrome-extension://naabfciimcjlmnohcligmgihhpihloah": {
+    "name": "IDmelon Accesskey",
+    "autoapprove": true
```

Comment:
Let's remove `"autoapprove": true` for now. (This key is a temporary workaround for the applications that are needed as secondary extensions in Kiosk mode, until we figure out a better solution.)